av1an-core/src/concat.rs

```diff
@@ -264,6 +220,19 @@ pub fn mkvmerge(
   Ok(())
 }
 
+/** This code adds the necessary command parameters to the options FILE
+ * that is passed in, up to the chunk NUM. Returns nothing.
+ */
+pub fn concatenate_mkvmerge(num: u64, mut this_file: File) {
+  let mut file_string: String = "[ \"-o\", \"output.mp4\", \"audio.mp3\", \"[\"".into();
+  for i in 0..num {
+    file_string.push_str(&format!(" , \"{:05}.ivf\" ", i));
+  }
+  file_string.push_str(",\"]\" ]");
+  println!("{}", file_string);
+  this_file.write_all(file_string.as_bytes());
```

Comment:
This needs a `.unwrap()` after it in order to ensure it flushes. (Rust's `write` functions are weird like that.)